client/landing/gutenboarding/onboarding-block/style-preview/font-select.tsx

```diff
@@ -54,7 +55,9 @@ const FontSelect: React.FunctionComponent = () => {
 				<span className="style-preview__font-option-contents">{ defaultFontOption }</span>
 			</Button>
 			{ fontPairings.filter( fontPairingsFilter ).map( fontPair => {
-				const isSelected = fontPair === selectedFonts;
+				// Font pairs are objects, we need `isShallowEqual` as we can't guarantee referential equality
+				// (E.g. if `selectedFonts` is coming from persisted state)
+				const isSelected = !! selectedFonts && isShallowEqual( fontPair, selectedFonts );
```

Comment:
Is `!! selectedFonts` needed here?

Reply:
Yeah, unfortunately :slightly_frowning_face: TS was complaining that `selectedFonts` could be `undefined`, which `isShallowEqual` doesn't want.

Sounds like something we might want to fix upstream.